lib/modulesync.rb

```diff
@@ -88,7 +83,7 @@ def self.manage_file(puppet_module, filename, settings, options)
     namespace = settings.additional_settings[:namespace]
     module_name = settings.additional_settings[:puppet_module]
     configs = settings.build_file_configs(filename)
-    target_file = module_file(puppet_module, filename)
+    target_file = puppet_module.path filename
```

Comment:
I prefer using braces to be explicit and I have the impression that most of the rest of the codebase does too.
```suggestion
    target_file = puppet_module.path(filename)
```

Reply:
Done. Thx